olive/passes/pytorch/sparsegpt.py

```diff
@@ -70,9 +70,9 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
                 description="Only prune layers whose name contains the given string(s).",
             ),
             # this is not the same as accelerator_spec.device which is the target device for inference
-            # compute_device is the device we want to run the algorithm on, does not affect the final model
-            # so accelerator_spec.device can be cpu but compute_device can be cuda for faster pass execution
-            "compute_device": PassConfigParam(
+            # device is the device we want to run the algorithm on, does not affect the final model
+            # so accelerator_spec.device can be cpu but device can be cuda for faster pass execution
+            "device": PassConfigParam(
```

Comment:
I forget what this device is? target_deivce(accelerators.device) or host_device? Or neither are? Could you remind me?

Looks like even we have target device and host device, we still cannot remove the `device` arguments for some particular passes.

Reply:
the compute_device should be host_device. I will let @jambayk to double confirm.